doc/source/developers_guide.rst

```diff
@@ -38,11 +38,9 @@ a GitHub account and then set to watch the repository at `GitHub Repository`_
 Requirements
 ------------
 
-    * Python_ 2.6, 2.7, 3.3-3.5
+    * Python_ 2.7, 3.4 or later
     * numpy_ >= 1.7.1
     * quantities_ >= 0.9.0
```

Comment:
I have lots of errors with nose test old version of wuantittes in an old env.
So we should force latest version here 0.12.1 

this is due to recent change in spiketrains:
```
    @property
    def times(self):
        return pq.Quantity(self)
```

Reply:
agreed, but let's update the requirements (which appear in several places) in a separate pull request